mast_aladin_lite/adapters/aladin_sync_adapter.py

```diff
@@ -0,0 +1,69 @@
+import astropy.units as u
+import numpy as np
+
+from astropy.coordinates import SkyCoord
+from IPython.display import display
+from mast_aladin_lite.app import gca
+
+from .viewer_sync_adapter import ViewerSyncAdapter
+
+
+class AladinSyncAdapter(ViewerSyncAdapter):
+    def __init__(self):
+        self.viewer = gca()
```

Comment:
I have mixed feeling about these global context variables.  I do see that they have value, especially in user convenience methods, but I think for building blocks like this, I'd rather see (at least the option of) specifying the `MastAladin` instance that this adapter is wrapping.  

Then, for example, a more general convenience method like `ViewerSyncUI` could be implemented to accept a list of viewers, or 2 `MastAladin` viewers, etc.  Not sure if those particular convenience methods would be useful, but it illustrates the additional flexibility of allowing the viewer arg in these constructors without really losing anything.